### PR TITLE
#28 Adding specification landing page

### PR DESCRIPTION
**Pull request by gunnarmorling**
@gsmet Quick one to prepare the website for the advent of the new revision.

### CODE REVIEW

File: specification/index.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+It's planned to be part of Java EE 8 (can of course be used with plain Java SE as the previous releases) and will primarly focus on leveraging the language features introduced in Java SE 8 for the purposes of validation.
+We'll also address other features further improving usefulness and usability of Bean Validation as requested by the community and as time allows.
+
+The JSR proposal can be found at [jcp.org](https://www.jcp.org/en/jsr/detail?id=380).
+The latest draft version of Bean Validation 2.0 can be found [here](/latest-draft).
+
+## Bean Validation 1.1
+
+Bean Validation 1.1 ([JSR 349](https://www.jcp.org/en/jsr/detail?id=349)) was finished in 2013 and is part of Java EE 7.
+It's main contributions are method-level validation, integration with CDI, group conversion and some more.
```
Comment (gsmet) [inline specification/index.md]: s/It's/Its/

Comment (gunnarmorling) [inline specification/index.md]: Agh, happens to me all the time :/

Comment (gunnarmorling) [inline specification/index.md]: Fixed.

**Comment by gunnarmorling** [comment]
@gsmet Force-pushed to fix that typo and added one more commit for #30.

**Comment by gsmet** [comment]
@gunnarmorling please don't force-push anything on this PR. I fixed a typo and the links locally and I'm ready to merge.

Just waiting for you to merge https://github.com/beanvalidation/beanvalidation-spec/pull/101 and deploy the draft before merging this one so that we don't have any broken link on the website.

Thanks.

**Comment by gsmet** [comment]
Rebased and merged with a few additional fixes.

Thanks!

**Pull request closed without merging**